.github/workflows/build-db.yml

```diff
@@ -44,37 +44,35 @@ jobs:
         run: |
           cd package/db
           FODB_MACHINE=arm64 FODB_SYSTEM=Darwin python -Im build
+          python -Im build --sdist
```

Comment:
Look, I don't like this answer either, but with the new v4 'immutable artifacts' rule I don't have any other choices to create an artifact that contains the wheel and the source file...

I will happily entertain other options if you have them!

Reply:
My assumption that the artifacts need to contain the source `tar.gz` and the whl are based on [the artifacts in this workflow](https://github.com/voxel51/fiftyone/actions/runs/9473030984) - if my assumption is wrong, that'd be swell, but 🤷 

This doesn't slow down the overall tests at all, the `build-db.yml` workflow still finishes under 2 minutes (with comparable times from 1m42s to 2m1s) and this isn't the long pole in the tent